chatbots-telegram-google-cloud-http-function/build.gradle.kts

```
@@ -0,0 +1,19 @@
+plugins {
+    id("io.micronaut.build.internal.chatbots-module")
+}
+
+dependencies {
+    implementation(mn.micronaut.serde.jackson)
+    api(mn.micronaut.gcp.function)
+    api(mn.google.function.framework)
+    api(project(":chatbots-telegram-core"))
+}
+
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        substitute(module("io.micronaut:micronaut-jackson-databind"))
+            .using(module("io.micronaut.serde:micronaut-serde-jackson:${mn.versions.micronaut.serialization.get()}"))
+    }
+}
+
+
```

Do we need to add

```suggestion
micronautBuild {
    sourceCompatibility = "11"
    targetCompatibility = "11"
}
```

In here?

Does that make this whole module Java 11+ only?  Or just this library?

---

not sure

---

ping @melix